sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -118,18 +164,69 @@ export class AppConfigurationClient {
     return this.client.deleteConfigurationSetting(key, options);
   }
 
+  /**
+   * Gets a setting from the Azure App Configuration service.
+   * 
+   * Example code:
+   * ```ts
+   * const setting = await client.getConfigurationSetting(key, { label: "MyLabel" });
+   * ```
+   * @param key The name of the key.
+   * @param options Optional parameters for the value being retrieved.
+   */
   getConfigurationSetting(key: string, options?: GetConfigurationSettingOptions): Promise<GetConfigurationSettingResponse> {
     return this.client.getConfigurationSetting(key, options);
   }
 
+  /**
+   * Lists settings from the Azure App Configuration service, optionally filtered by label, 
+   * accept date time or name.
+   * 
+   * Example code:
+   * ```ts
+   * // find all settings with label "MyLabel"
```

Comment:
Since the description above the example code talks about how the list method works, this comment inside the sample code is not required.